spring-kafka/src/test/java/org/springframework/kafka/retrytopic/RetryTopicConfigurationBuilderTests.java

```diff
@@ -107,6 +107,8 @@ void shouldSetNoBackoffPolicy() {
 	}
 
 	@Test
+	@Deprecated
+	@SuppressWarnings("removal")
```

Comment:
I think we should remove the test for deprecated API even now.

Reply:
I kept it as the deprecated code would be dead code and deprecating it doesn't mean nobody would call it. The test makes sure something else does not break it. Or do you mean removing the code and the test?